collections/_article/introducing-new-forum.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Introducing the new Godot Forum"
+excerpt: "We are excited to introduce a major improvement in terms of funding the development efforts of the Godot Engine."
+categories: ["news"]
+author: Winston Yallow
+image: /storage/blog/covers/introducing-new-forum.webp
+date: 2023-12-09 00:00:00
+---
+
+For the last couple of months, our Q&A platform was in readonly mode. Now it has been superceded by our new and more powerful forum! 🎉
+
+🔗 [https://forum.godotengine.org/](https://forum.godotengine.org/)
+
+And don't worry: no content is lost, all the old posts have been migrated to the new platform.
+
+## Why did we replace the old platform?
+
+The old Q&A platform was based on Question2Answer, an open source project for Q&A sites. Unfortunately it is only infrequently updated and had some security issues we couldn't easily solve. This led to problems like bot registrations and spam.
```

Comment:
```suggestion
The old Q&A platform was based on Question2Answer, an open source project for Q&A sites. Unfortunately it is infrequently updated and had some security issues we couldn't easily solve. This led to problems like bot registrations and spam which took a lot of manual effort to manage.
```

Two changes here:
- A grammar fix (removing "only")
- highlighting the human cost of dealing with bots and spam